### DISCUSSION TITLE
Local Build Option

### DISCUSSION THREAD

**Discussion by baxterblk** [RandomNinjaAtk/arr-scripts#211] [Q&A] [answered]
Do you have a build option?

**Comment by RandomNinjaAtk** [comment, marked as answer]
What do you mean by local build? 

These are just scripts, there isn’t anything to build….

**Comment by PrinceFrosty** [comment]
Create a Dockerfile that pulls the base image and runs the scripts. This is my `lscr.io/linuxserver/lidarr:extended` image Dockerfile
```Dockerfile
FROM lscr.io/linuxserver/lidarr:latest

RUN curl https://raw.githubusercontent.com/RandomNinjaAtk/arr-scripts/main/lidarr/setup.bash | bash
```
You could even place it inline [dockerfile_inline](https://docs.docker.com/reference/compose-file/build/#dockerfile_inline)
